Libraries/Boards/MAX32570/MN_EvKit_V1/Include/board.h

```diff
@@ -58,9 +62,6 @@ extern "C" {
 #define CONSOLE_BAUD 115200 /// Console baud rate
 #endif
 
-#define LED_OFF 1 /// Inactive state of LEDs
```

Comment:
As per of above explanation can be keep by 
Adding
#ifdef LED_OFF 
#undef LED_OFF
#endif      
#define LED_OFF 1

...

Reply:
https://github.com/Analog-Devices-MSDK/msdk/pull/613/commits/af7cbafb3edf54de1a9e6de86961bfb8c19e9d55